### ISSUE TITLE
Imagenet classification with deep convolutional neural networks

### ISSUE THREAD

**Issue by sooooner** [sooooner/papers#28]
# Imagenet classification with deep convolutional neural networks[[link](https://papers.nips.cc/paper/2012/hash/c399862d3b9d6b76c8436e924a68c45b-Abstract.html)]

**Alex Net**

## The Dataset
Image Net

##  The Architecture
![image](https://user-images.githubusercontent.com/31437443/112616337-f39ea180-8e66-11eb-8ea3-6cb27eaa5ab9.png)

###  Local Response Normalization
Relu를 사용하기 때문에 양수 output이 주변에 영향을 주는것을 방지하기 위해 feature map의 픽셀끼리 정규화 

![image](https://user-images.githubusercontent.com/31437443/112616775-77f12480-8e67-11eb-82ec-0184a8a7d4e7.png)

## Reducing Overfitting
1. Data Augmentation
2. Dropout
사용
